wdl2cwl/WdlV1_1ParserVisitor.py

```diff
@@ -4,12 +4,21 @@
     from .WdlV1_1Parser import WdlV1_1Parser
 else:
     from WdlV1_1Parser import WdlV1_1Parser
+from dataclasses import dataclass, field
+from typing import List
 
 # This class defines a complete generic visitor for a parse tree produced by WdlV1_1Parser.
 
+@dataclass
+class Struct:
+    name: str
+    fields: List[str] = field(default_factory=list)
```

Comment:
Happy to use something else if `dataclass` is not the best option here :+1:

Reply:
Ah! Py 3.6 doesn't have dataclasses, reverted it after CI failed.

Reply:
Well, Python 3.6 is EOL at the end of the year. And this is a pre-alpha codebase, so feel free to switch back and drop Python 3.6 from the testing matrix.